tests/main/classic-custom-device-reg/task.yaml

```diff
@@ -68,12 +68,13 @@ execute: |
     systemctl start snapd.service snapd.socket
 
     echo "Wait for seeding to be done"
-    while ! snap changes | grep -q "Done.*Initialize system state"; do sleep 1; done
+    retry -n 200 --wait 1 sh -c 'snap changes | MATCH "Done.*Initialize system state"'
```

Comment:
Can we use wait_for_first_boot_change here?

Reply:
done